package/csac_daemon/overlay/etc/init.d/S86csac_daemon

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+CSAC_PORT="/dev/ttyPS0"
+name="csac_daemon"
+cmd="csac_daemon --serial-port $CSAC_PORT"
+dir="/"
+log_tag=$name # to use logw
+source /etc/init.d/logging.sh # to use logw
+setup_loggers # to use logw
+user="portsd" #use ports daemon to share UART0
+
+setup_permissions() {
+    # Force device ownership
+    chown $user:$user ${CSAC_PORT}*
+}
+
+# make sure that we have a CSAC license file
+if [ -f /persistent/licenses/csac_license ]; then
```

Comment:
FYI, we've also been using the `/persisten/flags` directory to enable/disabling features that shouldn't be directly exposed to users.